src/twisted/application/twist/_twist.py

```diff
@@ -84,39 +84,35 @@ def startService(reactor, service):
 
 
     @staticmethod
-    def runnerOptions(twistOptions):
+    def runnerArguments(twistOptions):
```

Comment:
This is an artifact of the old RunnerOptions hooey.  I agree it should go away, but I haven't started cleaning up `twist` yet.  After I get the underlying `runner` API fixed up, that's next.